scripts/build_release_archive.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
```

Comment:
We should also delete this script from its old location.

Reply:
Done